onnx/version_converter/adapters/argmax_argmin_12_11.h

```diff
@@ -0,0 +1,26 @@
+// Adapter for ArgMax/ArgMin in default domain from version 12 to 11
+
+#pragma once
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+class ArgMaxArgMin_12_11 final : public Adapter {
+  public:
+    explicit ArgMaxArgMin_12_11(const std::string& op_name)
+      : Adapter(op_name, OpSetID(12), OpSetID(11)) {}
+
+    void adapt_argmax_argmin_12_11(std::shared_ptr<Graph> graph, Node* node) const {
```

Comment:
@askhade one job is still failing because `graph` is unused here, which triggers a warning treated as error: https://travis-ci.org/github/onnx/onnx/jobs/713539738#L2049
The problem is, if I remove the `graph` argument here I get the same exact warning for `adapt(std::shared_ptr<Graph> graph, Node* node)` a few lines below, and that can't be changed because that's how adapters are supposed to be called by the converter: https://github.com/onnx/onnx/blob/master/onnx/version_converter/convert.cc#L98

So I don't know how to solve this.

Reply:
you can simply change this to : 
void adapt_argmax_argmin_12_11(std::shared_ptr<Graph> , Node* node) const {

Reply:
Ah of course, unnamed arguments! Done